docs/user_guide/tutorials/train_peagle_offline.md

```
@@ -0,0 +1,379 @@
+
+**Parameters explained:**
+
+- `--model` - The target model you want to accelerate
+- `--data` - Dataset to use (built-in support for `sharegpt`, `ultrachat`, or a custom path to a jsonl file)
+- `--output` - Where to save preprocessed data
+- `--max-samples` - Limit samples (optional, good for testing/getting started)
+- `--seq-length` - Maximum sequence length
+
+**Expected output:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)